addons/portal/controllers/mail.py

```diff
@@ -113,11 +113,11 @@ def _portal_post_has_content(self, res_model, res_id, message, attachment_ids=No
         return bool(message) or bool(attachment_ids)
 
     @http.route('/mail/avatar/mail.message/<int:res_id>/author_avatar/<int:width>x<int:height>', type='http', auth='public')
-    def portal_avatar(self, res_id=None, height=50, width=50, access_token=None):
+    def portal_avatar(self, res_id=None, height=50, width=50, access_token=None, _hash=None, pid=None):
```

Comment:
Using a leading `_` in a variable/attribute/method/class name indicates that it is private which isn't the case here. I see that the function `_check_special_access` already uses this (bad) name and I care more about consistency than blindly following pep8 style.

The actual convention for naming a thing that would clash with a python builtin (here the `hash()` standard function) is actually to add a *leading* underscore.

Don't change anything, as I said consistency > conventions, but please don't think this is a normal thing to do :grimacing: